license.md

```diff
@@ -1,5 +1,6 @@
 MIT License
 
+Copyright (c) 2020 Vercel, Inc.
```

Comment:
? We are not employees of Vercel, and thus we have no authority to attribute copyright to them. Why is this here?

Reply:
I think it should be there it's just missing, I'm not an employees of them but AFAIK they have acquired all pkg stuff so the copyright is their. Betteer to ask @leerob to be sure

Reply:
We didn't acquire it - we created it! This was before my time at Vercel, but yes this should be MIT Copyrighted to the Vercel GitHub org. Also... I know 2020 was a blur but it's 2021 now 😄

Reply:
They will do it themselves. I’d rather not do someone else’s job, especially when I have absolutely no legal authority to do so.

Reply:
To clarify:

I know this could be hard to understand in this context because `pkg` is such an awesome and useful project. But the fact is that copyright can’t not be unilaterally attributed to someone, and we can not sign a license for Vercel. 

To make things easier to understand, imagine that someone wrote a malware or other unwanted softwares, and then put `Copyright acme inc.` in the executables.